mods/parsoid.js

```diff
@@ -809,6 +852,37 @@ module.exports = function(options) {
                 body: {
                     valueType: 'json'
                 }
+            },
+            // stashing resources for HTML, wikitext and data-parsoid
+            {
+                uri: '/{domain}/sys/key_rev_value/parsoid.stash.html',
+                body: {
+                    revisionRetentionPolicy: {
+                        type: 'latest',
+                        count: 1,
+                        grace_ttl: 86400
+                    },
+                    valueType: 'blob',
+                    version: 1
+                }
+            },
+            {
+                uri: '/{domain}/sys/key_rev_value/parsoid.stash.wikitext',
+                body: {
+                    valueType: 'blob'
+                }
+            },
+            {
+                uri: '/{domain}/sys/key_rev_value/parsoid.stash.data-parsoid',
+                body: {
+                    revisionRetentionPolicy: {
+                        type: 'latest',
+                        count: 1,
```

Comment:
As I understand, we don't need to store even 1 revision permanently, but set a `grace_ttl` immediately. May be we could try to use `count: 0`? I'm not sure that will work from the code perspective, we don't have any tests for this case.